docs/resources/networking_vip.md

```diff
@@ -9,47 +9,56 @@ Manages a VIP resource within HuaweiCloud. This is an alternative to `huaweiclou
 ## Example Usage
 
 ```hcl
-data "huaweicloud_vpc_subnet" "mynet" {
-  name = "subnet-default"
+resource "huaweicloud_vpc_subnet" "test" {
+  ...
 }
 
-resource "huaweicloud_networking_vip" "myvip" {
-  network_id = data.huaweicloud_vpc_subnet.mynet.id
+resource "huaweicloud_networking_vip" "test" {
+  network_id = data.huaweicloud_vpc_subnet.test.id
 }
 ```
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `region` - (Optional, String, ForceNew) The region in which to create the vip resource.
-  If omitted, the provider-level region will be used. Changing this creates a new resource.
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the network VIP.
+  If omitted, the provider-level region will be used. Changing this will create a new network VIP resource.
 
-* `network_id` - (Required, String, ForceNew) Specifies the ID of the network to which the vip belongs.
-  Changing this creates a new resource.
+* `network_id` - (Required, String, ForceNew) Specifies the ID of the network to which the network VIP belongs.
```

Comment:
better to make it clear that the network id is the subnet Network ID.

Reply:
yeah, the description should be clearer.